tests/test_crypto_utils.c

```diff
@@ -37,67 +38,60 @@ const char* device_id = "470010171566423";
 const uint8_t key[32] = {82,  142, 184, 64,  74, 105, 126, 65,  154, 116, 14,  193, 208, 41,  8,  115,
                          158, 252, 228, 160, 79, 5,   167, 185, 13,  159, 135, 113, 49,  209, 58, 68};
 
-int main(void) {
-  // TODO:prefer to use c unittest framework
+void setUp(void) {}
+
+void tearDown(void) {}
+
+void test_aes(void) {
   uint8_t ciphertext[1024] = {0}, iv[16] = {0}, plain[1024] = {0};
   uint32_t ciphertext_len = 0;
   /* mbedtls_aes needs r/w iv[] */
   uint8_t iv_en[16], iv_dec[16];
```

Comment:
Maybe we should use the macro defined in [utils/crypto_utils.h](https://github.com/DLTcollab/TA-endpoint/blob/develop/utils/crypto_utils.h#L18) size to replace the length of IV here.

Reply:
Done.